src/site/_includes/post.njk

```diff
@@ -1,9 +1,9 @@
 ---
-layout: default
+layout: default-next
```

Comment:
Let's not do that. We were supposed to only use it for /handbook for now.

Reply:
Yeh it's just there so people can see the articles for testing: we'll change before merge

Reply:
Haha lol you scared ppl with it:)